velox/exec/Window.cpp

```diff
@@ -41,16 +41,42 @@ void initKeyInfo(
   }
 }
 
-void checkDefaultWindowFrame(const core::WindowNode::Function& windowFunction) {
-  VELOX_CHECK_EQ(
-      windowFunction.frame.type, core::WindowNode::WindowType::kRange);
-  VELOX_CHECK_EQ(
-      windowFunction.frame.startType,
-      core::WindowNode::BoundType::kUnboundedPreceding);
-  VELOX_CHECK_EQ(
-      windowFunction.frame.endType, core::WindowNode::BoundType::kCurrentRow);
-  VELOX_CHECK_EQ(windowFunction.frame.startValue, nullptr);
-  VELOX_CHECK_EQ(windowFunction.frame.endValue, nullptr);
+// Update frameBound with the first row of the partition i.e zero for UNBOUNDED
+// PRECEDING frame bound.
+void findRowUnboundedPreceding(
```

Comment:
Since all these functions are one liners we needn't even abstract these and simply do the copies in the updateFrameBounds logic itself.

Reply:
Thanks for the suggestion, removed these functions!